src/OrchardCore.Modules/OrchardCore.Users/Views/UserFields.Edit.cshtml

```diff
@@ -72,15 +74,58 @@
 
                 element.type = 'password';
             }
+            @* The generated password should be at least 12 characters in length. But, it should meet the minimum requirements *@
+            const strongPasswordLength = @(Math.Max(12, Math.Max(IdentityOptions.Value.Password.RequiredLength, IdentityOptions.Value.Password.RequiredUniqueChars)));
```

Comment:
```
 const strongPasswordLength = @IdentityOptions.Value.Password.RequiredLength
```

Reply:
Math.Max ?

Reply:
@hishamco what @Skrypt is explained is correct.

This password generator generate a strong password that meets all the minimum requirements configurable in `IdentityOptions`. Do you see a specific issue?

Reply:
Just one comment why you are using `Math.Max()` here?

Reply:
Ok, I may agree that it restricts now to have generated passwords with 12 chars at least.
Let me check something first.

Reply:
RequiredUniqueChars defaults to 1.

https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.identity.passwordoptions.requireduniquechars?view=aspnetcore-6.0#remarks

RequiredLength defaults to 6.

https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.identity.passwordoptions.requiredlength?view=aspnetcore-6.0#remarks

We can remove the 12 chars fallback.

Reply:
A simple `iif` would work too now.

Reply:
Depends if the RequiredLength will be extended from server side if RequiredUniqueChars is higher. Then, if not, I agree with your requested change.

Reply:
@Skrypt @hishamco since this is a password generator, we should be creating a strong password (that meets the minimum requirement) instead of just a password that meets the requirements. I think we should generate a minimum of 10-12 characters password.

Reply:
Ok, I approved the PR. Unless someone requires eventually to be able to generate passwords that match their own custom requirements then this PR is fine. I let you guys decide if we enforce the tool to generate at least 12 chars. I would tend to not assume that it will be ok for everyone even if the password generated is not "strong" enough with only 6 chars for example.

Reply:
@Skrypt  if someone want to generate a weak password dispute best practices, then maybe they should manually write a password :)

Reply:
@hishamco @Skrypt @ns8482e I removed the min 12 characters requirement